etc/csp_whitelist.xml

```diff
@@ -7,33 +7,48 @@
                 <value id="fue-source" type="host">*.googleapis.com</value>
                 <value id="google-source" type="host">*.google.com</value>
                 <value id="gstatic-source" type="host">*.gstatic.com</value>
+                <value id="assets.adobedtm.com" type="host">assets.adobedtm.com</value>
+                <value id="cdn.cardknox.com" type="host">cdn.cardknox.com</value>
+                <value id="ajax.googleapis.com" type="host">ajax.googleapis.com</value>
+                <value id="js.braintreegateway.com" type="host">js.braintreegateway.com</value>

```

Comment:
Why is this needed?
Same for:
assets.adobedtm.com
web-sdk.aptrinsic.com

Reply:
@mstein-ck When I tested cardknox fields were not loaded, so I added those rules. 
but now I removed those rules, it works, so I removed it.
Thank you